File: C_Programming/Assignment/Assignment5/factorialfun.c
#include<stdio.h>
void isfact(int num);
void main()
{
    int num;
    printf("Enter the value of no.:");
    scanf("%d",&num);
    isfact(num);
//    printf("The factorial of no is:%d",x);
   
}
void isfact(int num)
{
    int fact=1;
    int temp=num;
    while(num)
    {
        fact=fact*num;
        num--;

    }
    
    printf("The factorial of %d is %d",temp,fact);
}

//output:Enter the value of no.:5
//The factorial of 5 is 120

File: Java_Programs/Assignment5Type(4)/Factorialfun.java
package Assignment5Type4;

public class Factorialfun {

	public static void main(String[] args) {
		int num=3;
//	    printf("Enter the value of no.:");
//	    scanf("%d",&num);
	   int x= isfact(num);
	   System.out.printf("The factorial of no is:%d",x);

	}
	
	public static int isfact(int num)
	{
	    int fact=1;
	    int temp=num;
	    while(num>0)
	    {
	        fact=fact*num;
	        num--;

	    }
	    
	    return fact ;
	}

}
